prep-css-flexbox/flex-wrap.html

```diff
@@ -0,0 +1,92 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <title>CSS Flexbox - flex-wrap</title>
+  <link rel="stylesheet" href="reset.css">
+  <link rel="stylesheet" href="starwars.css">
+  <style>
+/* your styles here 👇 */
+#card-deck {
+  display: flex;
+  flex-wrap: wrap;
+  flex-basis: calc(100% / 6);
```

Comment:
Try adding the `flex-basis` to another rule-set for `.card-container` instead of the whole `#card-deck`.

```css
.card-container {
  flex-basis: calc(100% / 6);
}
```